sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs

```diff
@@ -372,8 +371,8 @@ public async Task EventHub_InitialOffsetFromEnd()
                 });
             using (jobHost)
             {
-                // We don't expect to get signalled as there should be no messages received with a FromEnd initial offset
-                bool result = _eventWait.WaitOne(Timeout);
+                // We don't expect to get signaled as there should be no messages received with a FromEnd initial offset
+                bool result = _eventWait.WaitOne(1000);
```

Comment:
Why do we hardcode this one? What was Timeout?

Reply:
The flow basically says "wait for some period of time to make sure that we DON'T get something" then send something and verify that we get it.    The overall timeout that we apply for tests is several minutes; waiting for a second is enough to prove that nothing will be received.  We don't need to wait the full standard test timeout to confirm that something doesn't happen.

Reply:
Ah, would it make sense to wait 5 seconds just to make it more likely the test would fail  when it should?

Reply:
Shouldn't hurt.   Shifted to 5 seconds and added a constant to make the intent more clear.